### DISCUSSION TITLE
How to restore etcd backup from client node to master

### DISCUSSION THREAD

**Discussion by faheemabbas110** [etcd-io/etcd#15015] [Q&A]
I need guidance. I recently took the CKA exam, and there was a question about restoring the existing backup to etcd, but I was only logged in to the candidate node. Backup was placed on the candidate node, but when I tried to copy (scp) to the master node, it gave me a permission denied error.

Then I tried the following steps:

I switched the user to sudo -i

I copied the backup file using scp to the master node

I installed the etcdctl tool on the master because the ETCD_API etcdctl restore ... command was not installed there.

I executed the command to restore the database on path --data-dir /var/lib/etcd/backup

I updated the /etc/kubernetes/manifest/etcd.yaml --> path: /var/lib/etcd/backup

Even after doing all the steps, the cluster was not coming back. What should I do? Please guide me on how I can restore a backup from a client node to a master. It took a lot of time, and I wasn't able to complete all the questions.

**Comment by SimFG** [comment]
maybe `etcdutl` can help you, detail: https://github.com/etcd-io/etcd/tree/main/etcdutl
